packages/opentelemetry-web/src/utils.ts

```diff
@@ -127,6 +127,11 @@ export function getResource(
   >(),
   initiatorType?: string
 ): PerformanceResourceTimingInfo {
+  // de-relativize the URL before usage (does no harm to absolute URLs)
+  // Also, we mean the browser's URL API and not node's here.
+  // eslint-disable-next-line node/no-unsupported-features/node-builtins
+  spanUrl = new URL(spanUrl, location.href).toString();
```

Comment:
https://caniuse.com/#search=URL
IE doesn't support that, can you use the best and oldest parser for browsers :) ?
```
const a = document.createElement('a'); a.href = spanUrl;
spanUrl = a.href ;
```
I think it should be the same and for caching purposes you can create `a` element once in top of the file, WDYT ?

Reply:
Nice catch, thanks.  I didn't cache because I wasn't sure if the `a` would also cache the old location.href if it changes over the life of the page (e.g., SPA, history API usage, etc.).